### PR TITLE
update MASSCAP to include BTH and match ZD

### PR DESCRIPTION
**Pull request by combinatorist**

### CODE REVIEW

**Comment by combinatorist** [comment]
No idea why this PR is failing, especially after the second commit: https://github.com/codeforamerica/vita-min/pull/349/files/483756601338c801b05cc7137bf29dd3f3b1dbe2. I wonder if it won't let us rename groups here.

**Comment by combinatorist** [comment]
nm - we're using group name in the spec. I'll fix it.

File: db/vita_partners.yml [hunk 1]
```diff
@@ -220,51 +220,73 @@ vita_partners:
...
   zendesk_instance_domain: eitc
   zendesk_group_id: '360010415253'
   display_name: Massachusetts Association for Community Action (MASSCAP)
   source_parameters:
   - MC-WOR
   logo_path: ''
+- name: "[MA/BTH] ABCD"
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360011283393'
+  display_name: 'Boston Tax Help'
```
Comment (tdooner) [inline db/vita_partners.yml:271]: source paramater: BTHC-ABCD

File: db/vita_partners.yml [hunk 2]
```diff
@@ -220,51 +220,73 @@ vita_partners:
...
+  logo_path: ''
+- name: "[MA/BTH] AB APAC"
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360011284774'
+  display_name: 'Boston Tax Help'
```
Comment (tdooner) [inline db/vita_partners.yml:276]: source parameter: BTHC-AB-APAC

File: db/vita_partners.yml [hunk 3]
```diff
@@ -220,51 +220,73 @@ vita_partners:
...
+  logo_path: ''
+- name: "[MA/BTH] Roxbury & Jamaica Plain"
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360011284494'
+  display_name: 'Boston Tax Help'
```
Comment (tdooner) [inline db/vita_partners.yml:281]: source parameter: BTHC-ROXBURY

**Comment by tdooner** [comment]
Side note: when you merge this can you use the "Squash" option? I don't think the intermediate commits add much useful context in this case.

**Comment by combinatorist** [comment]
@tdooner, I actually removed the BTH source codes because Brian said they weren't ready to be routed (except StreetCred which wants it urgently). Doesn't this immediately turn them on? I suppose they could just avoid advertising those codes, but the cat may be out of the bag - IDK.

**Pull request merged**